client/components/Search.js

```diff
@@ -1,35 +1,147 @@
 import React from 'react'
 import elasticSearchClient from '../services/ElasticSearch'
+import Searchbar from './Searchbar'
+import SearchResult from './SearchResult'
 const { string } = React.PropTypes
+const numberOfResults = 12
 
-const Search = React.createClass({
-  propTypes: {
-    searchTerm: string
-  },
-  getInitialState () {
+export default class Search extends React.Component {
```

Comment:
Because I did not want to change all your work.. I can convert the other into ES6 Class as well if you want